### PR TITLE
Update the app:

### PR DESCRIPTION
**Pull request by yakutovicha**
Replace 'AiiDA Lab' with 'AiiDA lab'
Modify example notebook to work with the newest base widgets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [![Build Status](https://travis-ci.org/aiidalab/aiidalab-app-cutter.svg?branch=master)](https://travis-ci.org/aiidalab/aiidalab-app-cutter)
 
-# AiiDA Lab App cutter
+# AiiDA lab App Cutter
 
 Cookie cutter recipe for Applications running on
-[AiiDA Lab](http://aiidalab.materialscloud.org).
+[AiiDA lab](http://aiidalab.materialscloud.org).
```
Comment (ltalirz) [inline README.md]:
```suggestion
[AiiDA lab](https://www.materialscloud.org/aiidalab)
```

File: {{cookiecutter.repo_name}}/README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 ## Installation
 
 This jupyter-based app is intended to run in
-[AiiDA Lab](https://aiidalab.materialscloud.org)
+[AiiDA lab](https://aiidalab.materialscloud.org)
```
Comment (ltalirz) [inline {{cookiecutter.repo_name}}/README.md]:
```suggestion
[AiiDA lab](https://www.materialscloud.org/aiidalab)
```

**Review by ltalirz — COMMENTED** [review]
thanks for the update!
just two suggestions for the aiida lab url

**Comment by yakutovicha** [comment]
@ltalirz do you have an idea why can't it install `aiida-core=1.1.0`?

**Comment by ltalirz** [comment]
It's the python version  - still building on python 2.7

**Comment by ltalirz** [comment]
thanks @yakutovicha !

**Pull request merged**